Shared/Models/Message Model/Messageable.swift

```diff
@@ -84,4 +85,14 @@ extension Messageable {
             }
         }
     }
+
+    /// Returns the most recent reply to this message that is loaded or, if there are no replies, the message itself is returned.
+    /// NOTE: If this message has not loaded its replies yet, the most recent reply will not be available and nil will be returned.
+    var mostRecentMessage: Messageable? {
+        if self.totalReplyCount == 0 {
+            return self
+        }
+
+        return self.recentReplies.first
+    }
```

Comment:
I wasn't properly highlighting your first message in a thread because it's not included the replies array.

This is a convenient way to get the most recent message, including the message itself.

Reply:
Yeah I think I noticed a bug with the timestamps bc of this.